sw/device/silicon_creator/lib/error.h

```diff
@@ -51,12 +51,13 @@ enum module_ {
   kModuleAsn1 =            MODULE_CODE('A', '1'),
   kModuleRetRam =          MODULE_CODE('R', 'R'),
   kModuleXModem =          MODULE_CODE('X', 'M'),
-  kModuleRescue =          MODULE_CODE('R', 'S'),
+  kModuleRescue =          MODULE_CODE('R', 'C'),
```

Comment:
Why are we changing this?  There is existing code in the field with `RS` error constants.